.ado/windows-vs-pr.yml

```diff
@@ -243,6 +243,81 @@ jobs:
           workingDirectory: vnext
         condition: and(succeeded(), eq(variables['applyRnPatches'], 'true'))
 
+  - job: RNWSampleAppsPR
```

Comment:
Is it possible to make a template? There are other jobs like RNWUniversalOtherProjectsPR seems to share the same steps.

Reply:
Potentially, but I'm not going to do that until we switch to one toolchain and VM setup (#4101), to have a solid base. Otherwise each of these jobs has specific oddities that will need to be factored out.